### ISSUE TITLE
No vtrust-flash found after  executing ./start_flash.sh

### ISSUE THREAD

**Issue by klrock** [codetheweb/tuyapi#146]
**Describe the bug**

**To Reproduce**

**Expected behavior**

**Debug Output**
Post the output of your program/app/script when run with the `DEBUG` environment variable set to `*`.  Example: `DEBUG=* node test.js`.  Copy the output and paste it below (in between the code fences):

```

```

**Screenshots**

**Desktop (please complete the following information):**
 - OS: [e.g. macOS]
 - OS Version [e.g. 22]
 - Node Version [output of `node -v`]

**Additional context**

**Comment by codetheweb** [comment]
Hey @klrock, not really sure what you're asking/reporting.  By the subject line it appears to be related to firmware flashing, which TuyAPI doesn't do.  Did you mean to open an issue [here](https://github.com/ct-Open-Source/tuya-convert)?

**Comment by klrock** [comment]
I followed the instructions on "https://github.com/ct-Open-Source/tuya-convert" up to the ./start_flash.sh command, but found no vtrust-flash on my iPhone wifi net work list.  I am using Raspberry Pi 3 Model B v 1.2. with Raspian Stretch installed and updated.
I booted the RPI with ethernet attached pers instructions and was able to launch the browser via the GUI. I then went to the above website and copied/pasted the commends to the RPI root.  All seemed to go well until not able to find vtrust-flash.

The RPI setup called for the name/PW for my wifi router and is connected to it.  Do I have to disconnect RPI from my router? I did that and now the network status shows:
eth0.Configured 192.168.1.105/24
and
wlan0 disassociated from ASUS
Should I disconnect the ethernet cable?

Any help greatly appreciated.
